webapp/src/routes/home/game/index.js

```diff
@@ -17,12 +21,35 @@ export class Game {
     this.htmlElement.append(this._app.renderer.view);
 
     this.background = new Background({ width: this.width, height: this.height });
-    this.warehouse = new Warehouse({ rendererWidth: this.width, rendererHeight: this.height, scale: 1/3 });
+    this.warehouse = new Warehouse({ rendererWidth: this.width, rendererHeight: this.height, scale: 1 / 3 });
 
     this.stage.addChild(this.background.stage);
     this.stage.addChild(this.warehouse.stage);
+
+    this.displayFences().map(el => this.stage.addChild(el.stage));
   }
 
+  displayFences() {
```

Comment:
może zrobić osobną klasę `FenceGroup` albo coś w tym stylu i tam to przeliczyć i narysować? Lepiej jak najmniej trzymać w głównym pliku gry bo po chwili się śmietnik zaczyna robić